www/c-icap/src/opnsense/service/templates/OPNsense/CICAP/virus_scan.conf

```diff
@@ -0,0 +1,25 @@
+{% if helpers.exists('OPNsense.cicap.antivirus.enable_clamav') and OPNsense.cicap.antivirus.enable_clamav == '1' %}
+Service antivirus_module virus_scan.so
+ServiceAlias srv_clamav virus_scan
+ServiceAlias  avscan virus_scan?allow204=on&sizelimit=off&mode=simple
+virus_scan.ScanFileTypes TEXT DATA EXECUTABLE ARCHIVE GIF JPEG MSOFFICE
+
+{% if helpers.exists('OPNsense.cicap.antivirus.sendpercentdata') and OPNsense.cicap.antivirus.sendpercentdata != '' %}
+virus_scan.SendPercentData 5
+{% endif %}
+{% if helpers.exists('OPNsense.cicap.antivirus.startsendpercentdataafter') and OPNsense.cicap.antivirus.startsendpercentdataafter != '' %}
+virus_scan.StartSendPercentDataAfter 2M
+{% endif %}
+{% if helpers.exists('OPNsense.cicap.antivirus.allow204responses') and OPNsense.cicap.antivirus.allow204responses != '' %}
+virus_scan.Allow204Responces on
+{% endif %}
+{% if helpers.exists('OPNsense.cicap.antivirus.passonerror') and OPNsense.cicap.antivirus.passonerror != '' %}
+virus_scan.PassOnError off
+{% endif %}
+{% if helpers.exists('OPNsense.cicap.antivirus.cmaxobjectsize') and OPNsense.cicap.antivirus.maxobjectsize != '' %}
+virus_scan.MaxObjectSize  5M
+{% endif %}
+Module common clamd_mod.so
+clamd_mod.ClamdHost 127.0.0.1
+clamd_mod.ClamdPort 3310
```

Comment:
I started it by hand with debug level 9. It seems the BSD package doesn't recognize the socket variable:

```
Loading service :common path clamd_mod.so
Going to search variable ClamdSocket in table clamd_mod
Fatal error while parsing config file: "/usr/local/etc/c-icap/virus_scan.conf" line: 12
The line is: clamd_mod.ClamdSocket /var/run/clamav/clamd.sock

Fatal error while parsing config file: "/usr/local/etc/c-icap/c-icap.conf" line: 33
The line is: Include virus_scan.conf

Error opening/parsing config file

```

But it's identical with the line from the sample AND c-icap wiki.

I'll change the default for tcp option for clamav to enable=Y to avoid problems